### PR TITLE
Adding contact mail

### LINKED ISSUE #1400: Expose reference email address
Add a `<something>@qibo.science` email address to the contacts section in the README.

### PR DESCRIPTION
**Pull request by scarrazza**
Closes #1400.

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ To get in touch with the community and the developers, consider joining the Qibo
 
 [![Matrix](https://img.shields.io/matrix/qibo%3Amatrix.org?logo=matrix)](https://matrix.to/#/#qibo:matrix.org)
 
+If you have a question about the project, please [contact us with](mailto:qiboteam@qibo.science).
```
Comment (MatteoRobbiati) [inline README.md:64]:
```suggestion
If you have a question about the project, please contact as with [📫 qiboteam@qibo.science](mailto:qiboteam@qibo.science).
```

**Review by MatteoRobbiati — COMMENTED** [review]
Thanks. The email was not rendered in the README, a suggestion follows.

**Comment by scarrazza** [comment]
Please try to send an email so I can check this works.

**Comment by scarrazza** [comment]
@alecandido got it now, thanks.

**Pull request merged**